inventory/examples/crio/crio.inventory

```diff
@@ -0,0 +1,38 @@
+kube-master ansible_host=192.168.1.149
```

Comment:
I feel like we should change the default example inventories to use hostnames and not IP addresses. What do you think? Would make a cleaner example, and would work "out of the box" if using a non-host network (or a host network that has DNS).